website/versioned_sidebars/version-0.20-sidebars.json

```diff
@@ -7,7 +7,7 @@
       "version-0.20-accordproject-developers"
     ],
     "Getting Started": [
-      "version-0.20-started-installation",
```

Comment:
Same as above: this should be reverted, will be generated by docusaurus when we tag a new version.